GitUI/UserControls/FileStatusList.cs

```diff
@@ -697,64 +703,146 @@ public void SetDiffs(IReadOnlyList<GitRevision> revisions, Func<ObjectId, GitRev
                     if (conflicts.Count != 0)
                     {
                         // Create an artificial commit
-                        var desc = _combinedDiff.Text;
-                        tuples.Add((new GitRevision(ObjectId.CombinedDiffId), selectedRev, desc, conflicts));
+                        tuples.Add(new FileStatusWithDescription
+                        {
+                            FirstRev = new GitRevision(ObjectId.CombinedDiffId), SecondRev = selectedRev, Summary = _combinedDiff.Text, Statuses = conflicts
```

Comment:
```suggestion
                            FirstRev = new GitRevision(ObjectId.CombinedDiffId),
                            SecondRev = selectedRev,
                            Summary = _combinedDiff.Text,
                            Statuses = conflicts
```

Reply:
Applying this in SetStashDiffs() too, handling is uniform